src/devSVG.cpp

```diff
@@ -500,6 +500,20 @@ double svg_strwidth(const char *str, const pGEcontext gc, pDevDesc dd) {
                             gc->cex * gc->ps, is_bold(gc->fontface), is_italic(gc->fontface), file);
   FontMetric fm = gdtools::context_extents(svgd->cc, std::string(str));
 
```

Comment:
@hadley The problem with symbol font was getting too hairy (fixing it on the side of `svg_metric_info()` was breaking `svg_strwidth()`), so I changed the approach. I moved fallback computation from gdtools to svglite. In the end this makes the whole thing simpler I think.

Is this a reliable way of computing the length of the string? I check for Unicode with `mbcslocale`, and fallback to `strlen()` otherwise.